__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -303,7 +303,7 @@ exports[`linting typescript.ts fixture matches snapshot 1`] = `
     "line": 1,
     "message": "'unusedEs6Import' is defined but never used.",
     "messageId": "unusedVar",
-    "nodeType": "Identifier",
+    "nodeType": null,
```

Comment:
why did these change?

Reply:
I think this happens because of typescript-eslint/typescript-eslint#8640

I don't know the exact reasons behind this change, though.